app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
+
+  belongs_to :user
```

Comment:
- [ ] The ERD diagram shows that the user should be named `author` which should give our relationship foreign key ID as `author_id` and not the default `user_id`.

![author](https://user-images.githubusercontent.com/53124548/213439479-19ad0a45-a477-4745-bc54-dbaa3279b241.png)